sentences/en/scene_HassTurnOn.yaml

```diff
@@ -4,8 +4,17 @@ intents:
     data:
       - sentences:
           - "[activate|<turn>] <name> [scene] [on]"
+          - "(change|transition) to <name> [scene]"
         requires_context:
           domain: scene
         slots:
           domain: scene
         response: scene
+      - sentences:
+          - "[activate|<turn>] <area> <name> [scene] [on]"
+          - "[activate|<turn>] <name> [scene] (in|on|at) <area>"
```

Comment:
Could you please create a new common expansion rule for `(in|on|at)` called `<in>`? I think that'd be great to use for all intents.

Reply:
These changes are complete. I couldn't find any specific tests for expansion rules, but if I need to add some let me know where they are.